_layouts/product.html

```diff
@@ -91,8 +91,13 @@
 				<section class="product-info">
 					<h2>{{ product.name }}</h2>
 					<h4>
-						<a href="http://{{ product.url }}"target="blank">
-							{{ product.url }}
+						{% if product.url contains "http://" or product.url contains "https://" %}
```

Comment:
Yeah, but I didn't see much of a different in-practice, and I find this version to be nicely readable.)